### ISSUE TITLE
TypeError: Cannot read property 'then' of undefined

### ISSUE THREAD

**Issue by jonataswalker** [rollup/rollup#1532]
```
TypeError: Cannot read property 'then' of undefined
    at next (/var/www/FileBrowser/node_modules/rollup/dist/rollup.js:148:22)
    at /var/www/FileBrowser/node_modules/rollup/dist/rollup.js:152:48
    at <anonymous>
```

Rollup: `0.46.1`

This happens with `watch` option at the first modification.

**Comment by Rich-Harris** [comment]
Can you make a [repro](https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff) please?

**Comment by jonataswalker** [comment]
I'm trying but no success until now. Perhaps it's some plugin.

**Comment by jonataswalker** [comment]
@Rich-Harris actually the problem (my guess) is when you have multiple configs.

Repro: https://github.com/jonataswalker/rollup-watch-repro

**Comment by Rich-Harris** [comment]
Thank you — this is fixed in 0.47.5

**Comment by reactkeshav** [comment]
i Have also same problem.. please help me out

import React, { useState, useEffect } from 'react'
import { CCard, CCardHeader, CCardBody, CRow } from '@coreui/react'
import { Table, Tag, Space, Carousel, Popconfirm, message } from 'antd'
import '../../../scss/_custom.scss'
import DataBase from '../../../Firebase/DataFirebase'
import { Images } from '../../../Shared/Assets'
import ImageIcon from '../../../components/Atoms/ImageIcon'

import ProductDataSource from './Constant/ProductDataSource'

import '../BuisnessPost/style.scss'

// import Modal from 'src/components/Cell'17
import { Modal, Button, Form } from 'react-bootstrap'
import { object } from 'prop-types'

const BuisnessPost = () => {
  const { Column, ColumnGroup } = Table
  const [show, setShow] = useState(false)
  const [active, setActive] = useState()
  const [productsList, setProductsList] = useState(ProductDataSource)
  const [dataList, setDataList] = useState([])

  const handleClose = () => setShow(false)
  const handleShow = () => {
    setShow(true)
  }

  // delete handle
  function deleteHandle(ind) {
    setProductsList(productsList.filter((item) => item.key !== ind.key))
  }

  // data from firebase
  useEffect(() => {
    DataBase.getAllProducts().then((response) => {
      console.log('Data fetched', response)
      setDataList({
        dataList,
      })
    })
    //     .catch(() => {
    //       setDataList({
    //         ...dataList,
    //       })
    //     })
  })

  // message popups
  function confirm(e) {
    e.preventDefault()
    console.log(e)
    message.success('Click on Yes')
  }
  function cancel(e) {
    console.log(e)
    message.error('Click on No')
  }

  // const ProductDataSource = [
  //   {
  //     key: 0,
  //     Image: (
  //       <Carousel autoplay afterChange={onChange}>
  //         <div>
  //           <ImageIcon style={{ width: '100px' }} src={Images.AVATAR} />
  //         </div>
  //         <div>
  //           <ImageIcon style={{ width: '100px' }} src={Images.AVATAR} />
  //         </div>
  //       </Carousel>
  //     ),
  //     Product: 'cam',
  //     Price: ' $100 ',
  //     Description: ' Camera for photography ',
  //     status: 'Active',
  //     created: '12 june 2021',
  //   },
  // ]

  return (
    <>
      <CCard className="mb-4">
        <CCardHeader>
          Posts
          {/* <DocsLink href="https://coreui.io/docs/content/typography/" /> */}
        </CCardHeader>
        <CCardBody>
          <CRow>
            <Table dataSource={productsList}>
              <Column title="Image" dataIndex="image" key="Image" />
              <Column title="Product" dataIndex="product" key="product" />
              <Column title="Price" dataIndex="price" key="price" />
              <Column title="Description" dataIndex="description" key="description" />
              <Column title="Status" dataIndex="status" key="status" />
              <Column title="Created" dataIndex="created" key="created" />

              <Column
                title="Action"
                key="action"
                render={(text, record) => (
                  <Space size="middle">
                    <a className="View">View</a>
                    <a className="Approve" onClick={handleShow}>
                      Aproved status
                    </a>
                    <Popconfirm
                      title="Are you sure to delete this task?"
                      onCancel={() => {}}
                      onConfirm={confirm}
                      okText="Yes"
                      cancelText="No"
                    >
                      <a className="Delete">Delete</a>
                    </Popconfirm>
                  </Space>
                )}
              />
            </Table>
          </CRow>

          <Modal show={show}>
            <Modal.Header>
              <Modal.Title>Status</Modal.Title>
            </Modal.Header>
            <Modal.Body>
              <Form.Control
                as="select"
                value={active}
                onChange={(e) => {
                  e.preventDefault()
                  setActive(e.target.value)
                }}
              >
                <option value=""> Choose Type</option>
                <option value="Approved">Approved</option>
                <option value="Reject">Reject</option>
              </Form.Control>
            </Modal.Body>
            <Modal.Footer>
              <Button className="text-white" variant="secondary" onClick={handleClose}>
                Close
              </Button>
              {/* <Button className="text-white" variant="success" onClick={handleSubmitChange}>
                Save
              </Button> */}
            </Modal.Footer>
          </Modal>
        </CCardBody>
      </CCard>
    </>
  )
}
export default BuisnessPost
